angrmanagement/data/instance.py

```diff
@@ -208,6 +209,13 @@ def join_all_jobs(self):
         while self.jobs:
             time.sleep(0.05)
 
+    def append_code_to_console(self, hook_code_string):
+        console = self.workspace.view_manager.first_view_in_category("console") # type: ConsoleView
+        console.set_input_buffer(hook_code_string)
```

Comment:
Sorry to keep going back and forth with you on this, but I just realized this line has some problems - this will crash if there is no ConsoleView open. You probably want to add a `_get_or_create_console_view()` method to workspace to go alongside the similar methods for other views.